eng/code-quality-reports/src/main/resources/checkstyle/checkstyle.xml

```diff
@@ -378,6 +378,16 @@ page at http://checkstyle.sourceforge.net/config.html -->
     4) All classes should use ClientLogger as logger only but except ClientLogger itself -->
     <module name="com.azure.tools.checkstyle.checks.GoodLoggingCheck"/>
 
+    <!-- CUSTOM CHECKS -->
+    <!-- Public API classes should follow Camelcase rules: -->
+    <!-- 1. ClassName -->
+    <!-- 2. methodName -->
+    <!-- 3. variableName -->
+    <!-- 4. longerVariableOrMethodName -->
+    <!--<module name="com.azure.tools.checkstyle.checks.DisallowedWordsCheck">-->
+    <!--<property name="disallowedWords" value="URL, HTTP, XML" />-->
```

Comment:
URL, HTTP, XML, JSON, SAS, CPK

Reply:
So I have run the new checkstyle against the code and I probably fixed 100+ infractions against these words. In total I have 20ish files to commit. Did you want me to create a new branch for these changes or keep using this one?